packages/commercetools/theme/components/SideCart.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div id="cart">
+    <SfSidebar
+      :visible="isCartSidebarOpen"
```

Comment:
How are we planning to handle this outside of component, say i added to cart from another component somewhere down the layout tree, how do i dispatch that event and make sure this hears it @filrak ? In React for example, we could use state on the top level component and hook into it from both, or use redux. Whats the vue way?